pyedr/pyedr/tests/test_edr.py

```diff
@@ -212,6 +211,3 @@ def redirect_stderr(target):
     finally:
         sys.stderr = stderr
 
-
-if __name__ == '__main__':
-    unittest.main()
```

Comment:
Whilst I'm here, I'm going to remove this unittest section. Given it's all pytest, I'm not sure why it was here to begin with?

Reply:
good